pages/having-a-child.js

```diff
@@ -23,14 +24,101 @@ export default function lifejourney() {
       <div id="wb-bc" className="container pt-4">
         <Breadcrumb />
       </div>
-      <Accordion>
-        <div className="pr-6">
-          <MoreInfo text={t.moreInfoCommunity} />
-          <MoreInfo text={t.moreInfoFertility} />
-          <MoreInfo text={t.moreInfoHealth} />
-          <MoreInfo text={t.moreInfoFinancial} />
+
+      <div className="container flex flex-col md:flex-row align-items-center ">
+        <div className="pr-3 w-full lg:w-3/12 top-0 lg:sticky ">
```

Comment:
```suggestion
        <div className="pr-3 w-full lg:w-3/12 top-0 md:sticky ">
```
The side menu still floats to the left of the accordions at medium resolutions, maybe make it sticky at those resolutions as well? The menu container itself has a lot of extra space beside the 'quick links' at this res but I don't think that's something we need to worry a ton about at this stage.